pages/project-details/[id]/view-details/[viewId].js

```diff
@@ -19,17 +24,66 @@ export default function ViewDetails({
     );
   }
 
-  const { name, description, sketch } = currentView;
+  function handleDeleteImageViews(id) {
+    setViews(
+      views.map((view) =>
+        view.image.id === id
+          ? {
+              ...view,
+              image: [view.image].filter((image) => image.id !== id),
+            }
+          : view
+      )
+    );
+  }
+
+  async function handleImageChangeViews(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+
+    setStatusUpload("Loading...");
+
+    const response = await fetch("/api/upload", {
+      method: "post",
```

Comment:
i think "post" should be spelled in uppercase letters because in your ap/uplodi is the required method  is defined in uppercase

Reply:
I did the same as in the workshop 😅 i would leave it like that 🙂

Reply:
ok we learned it to make it uppercase ;)  but yeah its on you .. and its only a convention